test/elixir_analyzer/exercise_test/common_checks/private_helper_functions_test.exs

```diff
@@ -486,4 +490,96 @@ defmodule ElixirAnalyzer.ExerciseTest.CommonChecks.PrivateHelperFunctionsTest do
       assert PrivateHelperFunctions.run(code, @sqrt_example) == [{:fail, comment}]
     end
   end
+
+  describe "concept exercise with remote-control-car" do
+    # This exemplar has a default argument: def new(nickname \\ "none")
```

Comment:
Could we copy-paste the exemplar to `@remote_control_car_exemplar` instead of loading it from a file? If that happened around this line of code instead of at the very beginning of the file, we might not need this comment at all. The test will also continue working in case we ever change the exercise.